src/Orleans/Serialization/TypeUtilities.cs

```diff
@@ -217,19 +217,36 @@ public static string GetLocationSafe(this Assembly a)
             }
         }
 
-        public static bool IsTypeIsInaccessibleForSerialization(Type t, Module currentModule, Assembly grainAssembly)
+        public static bool IsTypeIsInaccessibleForSerialization(Type t, Module fromModule, Assembly fromAssembly)
         {
-            if(t.GetCustomAttributes(typeof(SerializableAttribute), false).Length > 0)
```

Comment:
Are you sure this check isn't needed anymore? I suspect, although I may be wrong, it was used to avoid generating serializers for types marked as [Serializable].